src/main/java/com/sda/zdjavapol107/checkers/CheckersBoard.java

```diff
@@ -0,0 +1,64 @@
+package com.sda.zdjavapol107.checkers;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+@Getter
+@Setter
+public class CheckersBoard {
+    private Pawn[][] checkersBoard;
+    private Map<Integer, Pawn> pawnMap;
+    private CheckersPlayer playerOne;
+    private CheckersPlayer playerTwo;
+    private CheckersTournament checkersTournament;
+    private CheckersTurn turn;
+
+
+    public CheckersBoard(CheckersPlayer playerOne, CheckersPlayer playerTwo) {
+        this.playerOne = playerOne;
+        this.playerTwo = playerTwo;
+        checkersBoard = generateNewBoard();
+        checkersTournament = new CheckersTournament();
+    }
+
+    public void printBoard() {
+        for (int i = 0; i < 8; i++) {
+            System.out.printf("%c %s\n", (char) 65 + i, Arrays.deepToString(checkersBoard[i]).replaceAll("null", "_"));
+        }
+        System.out.println("   1  2  3  4  5  6  7  8");
+    }
+
+    private Pawn[][] generateNewBoard() {
+        pawnMap = new HashMap<>();
+        for (int i = 0; i < 24; i++) {
```

Comment:
stale jak 24 albo nizej 8 staraj sie trzymac w zmiennych statycznych finalnych i wtedy je uzywac. gdyby jakims cudem ktorys z tych rozmiarow sie zmienil to zmieniasz to tylko w jednym miejsciu :) no i opisanie tego slownie np BOARD_SIZE, PAWN_AMOUNT czy cos w tym stylu jest czytelniejsze niz sama liczba ktora nic nie mowi